code/modules/clothing/under/miscellaneous.dm

```diff
@@ -96,21 +96,24 @@
 	item_color = "centcom"
 
 /obj/item/clothing/under/rank/centcom/officer
-	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant-Commander\" and bears \"N.C.V. Fearless CV-286\" on the left shoulder."
-	name = "\improper Nanotrasen Officers Uniform"
-	icon_state = "officer"
-	item_state = "g_suit"
-	item_color = "officer"
+	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant-Commander\" and bears \"N.A.S. Trurl \" on the left shoulder. Worn exclusively by officers of the Nanotrasen Navy. It's got exotic materials for protection."
+	name = "\improper Nanotrasen Naval Officer Uniform"
+	icon_state = "navy_gold"
+	item_state = "navy_gold"
+	item_color = "navy_gold"
 	displays_id = 0
 	flags_size = ONESIZEFITSALL
+	armor = list("melee" = 60, "bullet" = 60, "laser" = 60,"energy" = 60, "bomb" = 60, "bio" = 100, "rad" = 30, "fire" = 60, "acid" = 100)

```

Comment:
Not sure about the armor. Will this stack with specops gear, giving specops officers near-100 damage reduction? Unsure.